### PR TITLE
Added another ad-usergroup testcase.

### PR DESCRIPTION
**Pull request by kbidarkar**
*) Verify whether adding roles to user-group as well as to user
   for an external ad user is successful.

### CODE REVIEW

File: tests/foreman/ui/test_adusergroups.py [hunk 1]
```diff
@@ -368,7 +368,7 @@ def test_external_adusergroup_roles_update(self):
             self.assertIsNotNone(self.login.wait_until_element(
                 (strategy, value % self.ldap_user_name)
             ))
-            make_org(session, name=org_name)
+            make_org(session, org_name=org_name)
```
Comment (kbidarkar) [inline tests/foreman/ui/test_adusergroups.py]: This testcase was marked to be skipped, hence didn't find this bug earlier.

**Comment by kbidarkar** [comment]
This testcase passes.
```
(robottelo)[kbidarka@dhcp123 robottelo]$ nosetests -m test_external_adUserGroup_additional_user_role tests.foreman.ui.test_adusergroups
.
----------------------------------------------------------------------
Ran 1 test in 105.048s

OK
```

File: tests/foreman/ui/test_adusergroups.py [hunk 2]
```diff
@@ -464,9 +463,64 @@ def test_external_adUserGroup_additional_user_roles(self):
         UserGroup but those additional feature areas / roles assigned
         specifically to user
 
-        @status: Manual
-
         """
+        strategy, value = locators['login.loggedin']
+        foreman_role = gen_string("alpha")
```
Comment (oshtaier) [inline tests/foreman/ui/test_adusergroups.py]: Please, use single quote where you can

**Comment by rplevka** [comment]
ACK pending @oshtaier's (single quotes) comment

**Comment by sthirugn** [comment]
ACK pending open comments

File: tests/foreman/ui/test_adusergroups.py (hunk 1 above)
Comment (elyezer) [inline tests/foreman/ui/test_adusergroups.py]: :+1:

File: tests/foreman/ui/test_adusergroups.py (hunk 2 above)
Comment (JacobCallahan) [inline tests/foreman/ui/test_adusergroups.py]: :+1:

**Pull request merged**